controllers/vdoconfig_controller.go

```diff
@@ -787,6 +839,20 @@ func (r *VDOConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
 					}
 				}
 
+				return nil
+			})).
+		Watches(
+			&source.Kind{Type: &v1.ConfigMap{}},
+			handler.EnqueueRequestsFromMapFunc(func(object client.Object) []reconcile.Request {
+				configMap, _ := object.(*v1.ConfigMap)
```

Comment:
what happens if the configmap is deleted. should we unset the environment variables